src/ordered_hash_set.py

```diff
@@ -39,6 +45,11 @@ def remove(self, item):
 
         :param item: (object), Removed item
         """
+        index = 0
+        temp = self.get_all()
+        for i in range(self.__len__()):
+            if temp[i] == item:
+                index = i
```

Comment:
As I stated above use index(). The index method raises ValueError if the specified item is not in the list. Therefore you need to update docstring as well.